src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
```

Comment:
The convention is that class names should start with a capital letter: `PriorityQueue`